src/votekit/profile.py

```diff
@@ -12,9 +12,16 @@ class PreferenceProfile(BaseModel):
     candidates (list): list of candidates, can be user defined
     """
 
-    ballots: list[Ballot]
+    ballots: list[Ballot] = list()
     candidates: Optional[list] = None
 
+    def __eq__(self, other):
```

Comment:
this __eq__ method is pretty forgiving, PreferenceProfiles are equal to all non-PP objects and the only thing that's checked is that all ballots are in the other PP, not vice versa.  this could lead to some confusing behavior down the line

maybe add a few tests for equal/unequal PPs to flesh this out more